.github/workflows/deploy.yml

```
@@ -80,9 +109,71 @@ jobs:
-        with:
-          name: deployment-report
-          path: ./deployment-reports/
-          if-no-files-found: error
+
+  trigger-schema-parser:
+      needs: [deploy, build_schemas]
+      runs-on: ubuntu-latest
+      steps:
+        - name: Set Branch Based on Kernel
+          id: check-kernel
+          run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:28: Double quote to prevent globbing and word splitting

(shellcheck)